spec/datadog/appsec/waf/stress_test_spec.rb

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe 'stress tests', stress_tests: true do
```

Comment:
I would like to be specific here.

1. We are stress testing context here
2. describe still needs to highlight it
3. filename should also comply with it, we can use `context_stress_test`

All the steps above will prevent us from piling up code here and it will be very specific.

Reply:
Fixed in 7ccc0aeba56ee3a80c72fa056f86c85576afd276